[DOCS-1166][DOCS-880][DOCS-1167][DOCS-1168] Update automation docs (#1084)
wandb/docs: content/guides/integrations/yolov5.md

By passing a few simple command line arguments to YOLO, you can take advantage of even more W&B features.

> * Passing a number to `--save_period` will turn on [model versioning]({{< relref "/guides/models/registry/" >}}). At the end of every `save_period` epochs, saves the model weights to W&B and tags the best performing model on the validation set.

Reviewer: **suggestion:** 
```suggestion
* If you pass a number to `--save_period`, W&B saves a [model version]({{< relref "/guides/models/registry/" >}}) at the end of every `save_period` epochs. The model version includes the model weights and tags the best-performing model in the validation set.
```

Author: Thanks, I'll edit your suggestion to avoid the future tense. (It's a tech writing thing.)
